events/austria/vienna-organic-dancefloor.yaml

```diff
@@ -60,6 +60,20 @@ events:
       - Duo Härtel Wascher & Friends
     organisation: Simon Wascher
 
+  - name: Organic Dancefloor
+    links: []
```

Comment:
Is there some way people can get more details about this?

Reply:
I fear not. The event location doesn’t have it on their calender just yet … and [the general info about these events](http://organicdancefloor.tradmus.org/) had its last update in 2017.

I guess the URL will be https://fraumayer.at/event/haertel-wascher-duo-and-friends-organic-dancefloor-5 if they keep following the same pattern.